meteor-core/src/main/java/dev/pixelib/meteor/core/executor/ImplementationWrapper.java

```diff
@@ -16,7 +16,7 @@ public ImplementationWrapper(Object implementation, String namespace) {
         this.namespace = namespace;
     }
 
-    public <R> R invokeOn(InvocationDescriptor invocationDescriptor, Class<R> returnType /* not unused, see comment below */) throws NoSuchMethodException {
```

Comment:
`R` is a dynamic type which represents the expected type, and is used below to forcefully cast the returned type. This ensures that the actual returned value on the invoked method, at least inherits the signature of the invocation we expect. This means that we'll get an expected class cast exception here and means that we'll have some form of type safety, despite a generic `Object` being used elsewhere.

The *value* of `returnType` is indeed unused, but the type hint it gives isn't and it therefore is a valid reason to still include it in our opinion.

<img width="865" alt="image" src="https://github.com/pixelib/Meteor/assets/10709682/220b6701-839a-458a-b9fe-4693b56a9d71">